zeppelin-web/src/app/visualization/builtins/visualization-table.js

```diff
@@ -120,6 +125,18 @@ export default class TableVisualization extends Visualization {
           `,
           minWidth: this.getColumnMinWidth(colName),
           width: '*',
+          sortingAlgorithm: function(a, b, row1, row2, sortType, gridCol) {
+            const colType = gridCol.colDef.type.toLowerCase();
+            if (colType === 'number') {
```

Comment:
use TableColumnType.STRING
See TableColumnType def in `zeppelin-web/src/app/visualization/builtins/visualization-util.js`

Reply:
@r-kamath  address the comment and updated the PR as well.